### PR TITLE
[13.0][MIG] fieldservice_route

### PR DESCRIPTION
**Pull request by nikul-serpentcs**

### CODE REVIEW

**Comment by brian10048** [comment]
Thank you for working on this migration @nikul-serpentcs 

Any chance you can separate your changes specific to the migration task into separate commits?

For example,

previous 12.0 commits history
[IMP] module: black, isort, prettier
[MIG] module to 13

This will help us to review the changes more easily as the format changes show a lot of differences

**Review by dreispt — COMMENTED** [review]
@Nikul-Chaudhary The black, isort, etc changes should be in a separate commit from the manual changes. Can you do this please?

**Comment by nikul-serpentcs** [comment]
@dreispt Sure I will

File: fieldservice_route/migrations/12.0.2.0.0/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        "saturday",
+        "sunday",
+    ):
+        env.execute(
+            """UPDATE ir_model_data
+        SET name='fsm_route_day_%s'
+        WHERE name='fsm_route_day_%s' AND module='fieldservice_route';"""
+            % (index, day)
+        )
+        index += 1
```
Comment (brian10048) [inline fieldservice_route/migrations/12.0.2.0.0/pre-migration.py:25]: This file needs to be removed

**Review by brian10048 — CHANGES_REQUESTED** [review]

**Review by chienandalu — COMMENTED** [review]
Is this PR alive?

**Comment by chienandalu** [comment]
This changes aren't present in this migration: https://github.com/OCA/field-service/pull/742

**Pull request closed without merging**